### ISSUE TITLE
throttle all container content apps

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#4706]
**Describe the bug**

freezer contents overloads the VM and takes down Arctos

**To Reproduce**
1. Find a large container (eg freezer room),
2. Click on any 'contents' link
3. Scroll down to nothing, because you've just killed Arctos

**Expected behavior**

java memory or thread death errors are probably "expected" but that doesn't always happen and even those aren't entirely unfelt

**Additional context**

When https://github.com/ArctosDB/arctos/issues/4705 happened, there was very little traffic other than:

```
73.26.95.28 - - [22/May/2022:21:33:43 -0500] "GET /ContDet.cfm?container_id=18189039 HTTP/1.0" 200 733 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.4951.67 Safari/537.36"
```

I don't think there's any other possible explanation for the outage, which resembles previous outages

**Priority**

Critical; we can't have Arctos going down from a widely-available link.

**Comment by campmlc** [comment]
Yes, that's exactly what killed it last night. But I need some means of
getting at these data. It can be freezer by freezer. But I have to be able
to download container contents csv and see all collection objects in a
container. Happy to send list of containers I need this for and wait on a
list or link to avoid going through browser.

**Comment by dustymc** [comment]
>some means of
getting at these data

New issue, I need to know exactly what data you need.

**Comment by campmlc** [comment]
The info from show all collection objects in container ( with specimen
download) and the csv download of container contents. I'll file an issue
with specific barcodes, but for now the most important are
DGR19060
DGR12750
DGR12753
DGR12754
DGR12648
DGR12748
DGR12749 (note no rack positions)
DGR12751
